common/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ set (ASEBACOMMON_SRC
 	utils/BootloaderInterface.cpp
 	msg/msg.cpp
 	msg/NodesManager.cpp
+	zeroconf/zeroconf.cpp
```

Comment:
There is a missing check for `dns_sd.h` and the related system-dependent library to link.